content/statsServiceMap.md

```diff
@@ -1,16 +1,14 @@
 ---
 author: David Watkins
-date: 2016-06-23
+date: 2016-07-29
 slug: stats-service-map
 draft: True
 title: Using the dataRetrieval Stats Service
 categories: Data Science
 tags: 
   - R
-  - dataRetrieval
-image: static/stats-service-map/plot-1.png
```

Comment:
Want to add dataRetrieval and the path to the image back in? At some point, I'd like to update USGSmarkdowntemplates that we can specify them in there, but for now, we still need to do it by hand